From 42906f4bee5585605c1af40b376e232ac9b7036e Mon Sep 17 00:00:00 2001
From: Pierre-Hugues Husson <phh@phh.me>
Date: Tue, 11 Aug 2026 09:12:54 +0000
Subject: [PATCH] Include vendor.samsung.hardware.radio.network-V1-java in base
 libs, will be used by f/o/telephony

[Port to A16] Radio HAL libs bumped V3->V5 upstream; samsung lib inserted after android.hardware.radio.network-V5-java.

Change-Id: I642cca19f0de883dd41ddcf7ea258fe696349a67
---
 Android.bp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Android.bp b/Android.bp
index 939c2417a627..1ede4ccf8731 100644
--- a/Android.bp
+++ b/Android.bp
@@ -232,6 +232,7 @@ java_library {
         "android.hardware.radio.messaging-V5-java",
         "android.hardware.radio.modem-V5-java",
         "android.hardware.radio.network-V5-java",
+        "vendor.samsung.hardware.radio.network-V1-java",
         "android.hardware.radio.sim-V5-java",
         "android.hardware.radio.voice-V5-java",
         "android.hardware.thermal-V1.0-java-constants",
-- 
2.43.0

